### ISSUE TITLE
Support for the old SASS syntax?

### ISSUE THREAD

**Issue by claudiu-cristea** [scssphp/scssphp#92]
We are still using the old Ruby Sass, but installing it we're getting:

```
Ruby Sass has reached end-of-life and should no longer be used.
```

I like the idea of using scssphp instead, but I'm hit this error trying to compile:

```
./vendor/bin/pscss web/themes/joinup/prototype/sass/app.sass
PHP Fatal error:  Uncaught ScssPhp\ScssPhp\Exception\ParserException: parse error: failed at `@import "base/fontfaces"` app.sass on line 2, at column 1 in vendor/scssphp/scssphp/src/Parser.php:144
Stack trace:
#0 vendor/scssphp/scssphp/src/Parser.php(198): ScssPhp\ScssPhp\Parser->throwParseError()
#1 vendor/scssphp/scssphp/src/Compiler.php(260): ScssPhp\ScssPhp\Parser->parse('// Font declara...')
#2 vendor/scssphp/scssphp/bin/pscss(211): ScssPhp\ScssPhp\Compiler->compile('// Font declara...', 'app.sass')
#3 {main}
  thrown in vendor/scssphp/scssphp/src/Parser.php on line 144
```

The input file is this https://github.com/ec-europa/joinup-dev/blob/develop/web/themes/joinup/prototype/sass/app.sass

Any idea?

**Comment by Cerdic** [comment]
Hi, scssphp has no support for the original old sass syntax, only for the scss syntax.

The question if we should add this support for sass syntax is relevant as we're trying to comply with the full sass-lang specification - which of course includes sass syntax support.

But at the moment this is really low in the priority list.

I don't know what your worklfow and codebase volume are, but maybe you can switch to the scss syntax with a tool like https://www.npmjs.com/package/sass-convert ? (didn't try it)

**Comment by claudiu-cristea** [comment]
@Cerdic thank you. I guess there's no other PHP lib, out in the wild, knowing to compile the old SASS.

**Comment by claudiu-cristea** [comment]
I've tested https://www.npmjs.com/package/sass-convert and the results are amazing. It does a 100% conversion. Then I've compared the compiled CSS files, one with the old Ruby SASS and the other using SCSS files, converted from SASS with https://www.npmjs.com/package/sass-convert, and compiled with SCSSPHP. There are no differences, except some that are not impacting the results, such as `opacity:.5;` vs. `opacity:0.5;`. Thank you for the hint!
